src/Toolkit/Toolkit/UI/Controls/PopupViewer/PopupMediaView.cs

```diff
@@ -102,11 +110,15 @@ private void UpdateImage()
                     img.GestureRecognizers.Add(tapGesture);
 #else
                     img.Tag = linkUrl;
+#if WPF
                     if(img.Cursor != Cursors.Hand)
                     {
                         img.Cursor = Cursors.Hand;
                         img.MouseLeftButtonDown += (s, e) => _ = Launcher.LaunchUriAsync((s as Image)?.Tag as Uri);
                     }
+#elif WINDOWS_XAML
+                        img.Tapped += (s, e) => _ = Launcher.LaunchUriAsync((s as Image)?.Tag as Uri);
```

Comment:
If you use the previous/next buttons in the media gallery, and then tap one of the images, it opens MANY tabs of the same link at once.  Are these event handlers getting stacked somehow?